demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml

```diff
@@ -99,7 +107,8 @@ spec:
                       cpu: 2
                       memory: 8G
                       nvidia.com/gpu: 0
-          rayVersion: 1.12.0
+                imagePullSecrets: []
+          rayVersion: 2.1.0
```

Comment:
this could be set to `2.5.0` But I also don't think this field is used by anything.

Reply:
I've created an issue to track this

Reply:
https://github.com/project-codeflare/codeflare-sdk/issues/299